### ISSUE TITLE
Committee endpoint not returning reports and financials for Unauthorized, Party - Nonqualified committees

### ISSUE THREAD

**Issue by apburnes** [fecgov/openFEC#3382]
When looking into reports or financial summaries for committees that are `Party Nonqualified`, no results are returned to be displayed.  API is not finding or parsing this committee type.

See http://localhost:8000/data/committees/?committee_type=X to find above committee types.

Initial Reference Issue: https://github.com/fecgov/fec-cms/issues/1745

**Comment by jwchumley** [comment]
This is almost certainly the same problem Helen is working on for committees that file the wrong form type. In this case though they are filing correctly but the api doesn't account for inaugural or host committees which are PARTY committees (committee type x or y). THe API code thinks all X or Y committees file form 3x but these file form 13 or form 4. Helen's view will have to include form 13 and form 4 along with 3, 3x and 3p. @hcaofec @PaulClark2 @AmyKort @pkfec

**Comment by lbeaufort** [comment]
@jwchumley 

It looks like the committee page totals are coming from https://fec-dev-api.app.cloud.gov/v1/committee/C00629584/totals/ which uses the `TotalsCommitteeView` resource which has a list of models in the `totals_schema_map` that doesn't have a lookup for this committee type, so it falls back to `default_schemas` which points to `models.CommitteeTotalsPacParty`. 

This model (`models.CommitteeTotalsPacParty`) points to `ofec_totals_pacs_parties_mv` which joins `ofec_totals_combined_mv` and `ofec_committee_detail_mv` where `form_type` is `F3X`. It looks like `ofec_totals_combined_mv` has an entry for our example C00629584 but it has `form_type` F13 - maybe we just add F13 to the default model (`ofec_totals_pacs_parties_mv`) or add a new MV and model to the `default_schemas` for committee type X.

**Comment by PaulClark2** [comment]
The primary focus of this ticket should be inaugural committees (Form 13 filers; committee type "X").  If adding the form type to the default model resolves this issue, we could also include/test convention and host committees (Form 4/3X filers; committee types "X," "Y" or "N").

**Comment by PaulClark2** [comment]
Just noting this here, it looks like we have a similar problem with Form 9 filers. You can find F9s on the filings page but not on the "committee's" profile page. 

Filings page
https://www.fec.gov/data/filings/?data_type=processed&committee_id=C30001689&form_type=F9

Committee profile page
https://www.fec.gov/data/committee/C30001689/?tab=filings

**Comment by pkfec** [comment]
committee types are mapped to a MV here:
https://github.com/fecgov/openFEC/blob/develop/webservices/resources/totals.py#L15-L23.
However SQL is filtering the committees only of form type F3X, which is why we are not getting the expected results on the committee pages. 

As a part of refactoring this code, should we consider adding other missing committee's from the MAP and add or update the SQL to include the respective forms?

**Comment by pkfec** [comment]
While adding F13's to the **ofec_totals_pacs_parties_mv**  noticed that there are 4 MV's that need to be created/dropped and renamed as part of the MV life cycle . It seems more work than expected and also there is no F13 test data in the **sample_db.sql** file. 

**1. ofec_totals_pacs_parties_mv
2. ofec_totals_pacs_mv
3. ofec_totals_parties_mv
4. ofec_entity_chart_mv**

1. @hcaofec mentioned dropping the `ofec_entity_chart_mv` as this is no longer needed. And `ofec_entity_chart_mv` is used in `ofec_totals_pacs_parties_mv`. Once `ofec_entity_chart_mv` is dropped, i have to remove this MV from the V106 migration. i am slightly blocked on this.

2. I am going to create a new ticket and assign it to the the DBA team asking them create test data for 
    F13 and update the script and **sample_db.sql**. Without inserting the test data into the base tables,
   cannot test the F13 fix. I am blocked on this.

cc @hcaofec @fecjjeng @ChingKuan @jwchumley

**Comment by dorothyyeager** [comment]
Moving forward to 7.3, as work continues on the related issues.

**Comment by fecjjeng** [comment]
Migration file added.  The definition of public.ofec_totals_pacs_parties_mv had been updated to include F4 and F13.  The two depending MVs that only filter on committee_type (public.ofec_totals_pacs_mv, ofec_totals_parties_mv) had been replace with view only.  The corresponding model had been updated to use the views in totals.py.  The manage.py had been updated too to remove the MV refresh of these two dropped views.  I was considering to replace ofec_totals_pacs_parties_mv with a view too.  But since the underlining MVs does not have indexes added, I will keep ofec_totals_pacs_parties_mv as a MV at this moment.  Will revisit.

Sample data had added for F13 and F4.
If one run createSampleDB then in psql run the query
select * from ofec_totals_pacs_parties_mv order by idx;
You should see 2 rows for F13 and two rows for F4.
"C00458166";2010;"F13"
"C00540005";2014;"F13"
"C00493254";2016;"F4"
"C00493254";2018;"F4"

**Comment by pkfec** [comment]
@AmyKort @jwchumley i have updated the completion criteria above.

**Comment by dorothyyeager** [comment]
Waiting on some API and front-end changes to merge.
